stable/memcached/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 1.0.0
+version: 2.0.0
```

Comment:
@zoidbergwill yeah I think it's fine for this to be just a minor bump, good point though